From b14b770573f46674c8a4319a4160d1ae01dbdc2d Mon Sep 17 00:00:00 2001
Message-Id: <b14b770573f46674c8a4319a4160d1ae01dbdc2d.1376408877.git.minovotn@redhat.com>
In-Reply-To: <823d94e906b2aeecd2c60bc9f19ca8948a756f27.1376408877.git.minovotn@redhat.com>
References: <823d94e906b2aeecd2c60bc9f19ca8948a756f27.1376408877.git.minovotn@redhat.com>
From: Gerd Hoffmann <kraxel@redhat.com>
Date: Tue, 30 Jul 2013 11:35:23 +0200
Subject: [PATCH 05/18] qemu-char: print notification to stderr

RH-Author: Gerd Hoffmann <kraxel@redhat.com>
Message-id: <1375184130-11076-6-git-send-email-kraxel@redhat.com>
Patchwork-id: 52784
O-Subject: [RHEL-6.5 qemu-kvm PATCH 05/12] qemu-char: print notification to stderr
Bugzilla: 676568
RH-Acked-by: Laszlo Ersek <lersek@redhat.com>
RH-Acked-by: Luiz Capitulino <lcapitulino@redhat.com>
RH-Acked-by: Michael S. Tsirkin <mst@redhat.com>

Signed-off-by: Gerd Hoffmann <kraxel@redhat.com>
Reviewed-by: Laszlo Ersek <lersek@redhat.com>
Signed-off-by: Michael Tokarev <mjt@tls.msk.ru>
(cherry picked from commit fdca2124adc293f84f2b7aaf0df43faa6b6bf420)
---
 qemu-char.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

Signed-off-by: Michal Novotny <minovotn@redhat.com>
---
 qemu-char.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/qemu-char.c b/qemu-char.c
index d3470bb..1a5f34a 100644
--- a/qemu-char.c
+++ b/qemu-char.c
@@ -2489,8 +2489,8 @@ static CharDriverState *qemu_chr_open_socket_fd(int fd, bool do_nodelay,
     }
 
     if (is_listen && is_waitconnect) {
-        printf("QEMU waiting for connection on: %s\n",
-               chr->filename);
+        fprintf(stderr, "QEMU waiting for connection on: %s\n",
+                chr->filename);
         tcp_chr_accept(s->listen_chan, G_IO_IN, chr);
         socket_set_nonblock(s->listen_fd);
     }
-- 
1.7.11.7

